modules/representation_management/app/controllers/representation_management/v0/pdf_generator_base_controller.rb

```diff
@@ -48,6 +48,7 @@ def veteran_params_permitted
       end
 
       def flatten_claimant_params(claimant_params)
```

Comment:
Could you create 2 new variable in `flatten_claimant_params`

```ruby
claimant = claimant_params.dig(:claimant)
address = claimant.dig(:address)
```
then you can do: 
```ruby
claimant_first_name: claimant.dig(:name, :first)
claimant_city: address.dig(:city)
```
It would make it much easier to read

Reply:
Done!  This reduced some calls to `claimant.dig(:date_of_birth)` which triggered additional `Style/SingleArgumentDig` rubocop warnings.  Those were fixed by converting those to `claimant[:date_of_birth]`.

Reply:
I didn't realize there was a cop for that, but that's good know thanks